Placeeo/Controller/LocationManager.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  LocationManager.swift
+//  Placeeo
+//
+//  Created by Pasha Berger on 03.10.2021.
+//
+
+import Foundation
+import CoreLocation
+
+class LocationManager: NSObject, CLLocationManagerDelegate {
+    static let shared = LocationManager()
+    
+    let manager = CLLocationManager()
+    
+    var completion: ((CLLocation) -> Void)?
```

Comment:
did you make up this solution by yourself, or found it?

Reply:
i found it on youtube